lib/immobilienscout/parsers/xml.rb

```diff
@@ -0,0 +1,11 @@
+module Immobilienscout
+  module Parsers
+    class Xml < ResponseParser
+      def parse
+        response_body = JSON.parse(Hash.from_xml(@response.body).to_json)
```

Comment:
@padde mentioned it in the last review already: Maybe I missed something while I was on vacation, but why do we use `to_json` and then `JSON.parse`? It seems like `Hash.from_xml` is already giving us the hash we need, no?

Reply:
Yes I know, but this is needed, I was going to explain it in the description but if I don't add the parse again is returning 🤷‍♀ @padde

Reply:
What kind of error?